packages/amplication-data-service-generator/src/update-package-jsons.ts

```diff
@@ -49,16 +49,16 @@ function updatePackageJSON(module: Module, update: Record<string, any>) {
  */
 function updatePackageLockJSON(
   module: Module,
-  update: Record<string, any>
+  update: { [key: string]: any }[]
 ): Module {
   const lockfile = JSON.parse(module.code);
   /**
    * The v2 lockfile contains a package in the packages section that is a mirror of the project so we need to update the static data there also
    */
   const pkg = lockfile.packages[""];
 
-  Object.assign(lockfile, update);
-  Object.assign(pkg, update);
+  Object.assign(lockfile, ...update);
+  Object.assign(pkg, ...update);
```

Comment:
I know this is not part of this PR but still asking - is there a reason we use object.assign instead of merge like in the other places ?

Reply:
I think it's by mistake. I'll fix it